### DISCUSSION TITLE
duplicate reads

### DISCUSSION THREAD

**Discussion by INFINITY1993** [metagenome-atlas/atlas#500] [Q&A]
Hi, dear developer,
I noticed that you integrated reads deduplication in the QC step. What I wanna ask is that is it really necessary to do deduplication? what are the benefits or drawbacks? Thanks in advance!
Best regards!

**Comment by SilasK** [comment]
Many metagenoics libraries are prepared with a small amplification step.  Therefore there might be some reads that are artificial duplicates. 


This step removes them it also sorts and compresses the reads so that the dastq gz file is up to 30% smaller, according to the claim. 
I thought that this could be a good step to implement. But you can easily deactivate it in the config file. 

Do you have a reason for nod-removing the deduplicaded reads?

In my experience the deduplicaded reads are only an tiny fraction. But if you have low biomass samples they might be higher. 


I suggest you to see also the docs of the tool in question, [clumpify]( https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/clumpify-guide/)

**Reply by INFINITY1993** [reply]
Thanks for your quick response and detailed explanation.
I have compared the assembly results (deduplication or not), and the results are quite similar. I just want to seek for more explanations.
what I also want ask is that: will the error correction tool tadpole be faster than the tools inside metaspades?

**Reply by SilasK** [reply]
I don't know about a specific benchmark.
I expect taedpole to be faster and more memory efficient. But maybe a bit less sensitive.

**Reply by INFINITY1993** [reply]
I see. Thanks so much! 
ATLAS is really a good pipeline, which is more flexible. We are installing it in our laboratory server.
